tests/stdlib/algorithm/tgraph.nim

```diff
@@ -0,0 +1,101 @@
+import std/unittest
+import experimental/graph
+import std/osproc
+import std/sets
+import std/algorithm
+import std/sequtils
+
+type Id = int
+
+proc arrangeRepr(graph: HGraph[Id]) =
```

Comment:
What's this for?

Reply:
For debugging, I will move it to tests later on, `graph-easy` allows to layout the graph in terminal from the textual description. Graphviz repr could be added as well, debugging graph data structures and algorithms usually involves some kind of visualization, very often, so I decided to include this as well.